packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-base.spec.tsx

```diff
@@ -0,0 +1,221 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { Components } from '../../../components.d';
+import { CustomHTMLElement } from '../../../utils/types';
+import { ScSizeProvider } from '../../sc-size-provider/sc-size-provider';
+import { ScGridTooltip } from '../../sc-widget-grid/sc-grid-tooltip';
+import { ScWidgetGrid } from '../../sc-widget-grid/sc-widget-grid';
+import { ScDialBase } from './sc-dial-base';
+import { update } from '../../charts/common/tests/merge';
+import { DEFAULT_CHART_CONFIG } from '../../charts/sc-webgl-base-chart/chartDefaults';
+import { MINUTE_IN_MS } from '../../../utils/time';
+import { DATA_STREAM, NUMBER_INFO_1, NUMBER_STREAM_1 } from '../../../testing/__mocks__/mockWidgetProperties';
+import { round } from '../../../utils/number';
+import { DataType } from '../../../utils/dataConstants';
+import { COMPARISON_OPERATOR, StatusIcon } from '../../charts/common/constants';
+import { StatusProgress } from './util';
+
+const VIEWPORT = {
+  ...DEFAULT_CHART_CONFIG.viewport,
+  duration: MINUTE_IN_MS,
+  yMin: 0,
+  yMax: 5000,
+};
+
+const VALUE1 = 1250;
+const VALUE2 = 3250;
+const VALUE3 = 4500;
+
+const newValueSpecPage = async (propOverrides: Partial<Components.ScDialBase> = {}) => {
+  const page = await newSpecPage({
+    components: [ScDialBase, ScSizeProvider, ScWidgetGrid, ScGridTooltip],
```

Comment:
does this test require `ScWidgetGrid`?

Reply:
It's not required. And I removed `ScWidgetGrid `